### PR TITLE
Feature/35910 new govuk restarts update

### PR DESCRIPTION
**Pull request by GeorgeChatzigiannis**
- Moved pupil form under pupil register view directory
- Used session expiry component with updated styles on pupil register pages

- [x] Updated restarts overview page
- [x] Updated pupil restart select page

### CODE REVIEW

File: admin/views/pupil-register/pupils-list.ejs [hunk 1]
```diff
@@ -14,7 +14,7 @@
       <% } %>
       <%- partial('partials/_readonly', {availabilityData}) %>
       <h1 class="govuk-heading-xl">Pupil register</h1>
-      <%- partial('partials/_session_expiry') %>
+      <%- partial('partials/_gds_session_expiry') %>
```
Comment (jon-shipley) [inline admin/views/pupil-register/pupils-list.ejs]: Why is it called _gds_session_expiry?  This is not vendored code.

Comment (GeorgeChatzigiannis) [reply admin/views/pupil-register/pupils-list.ejs]: @jon-shipley it's the prefix I am using for any component that is using the new toolkit markup and styles. Happy to use another convention, any suggestions?

Comment (jon-shipley) [reply admin/views/pupil-register/pupils-list.ejs]: _mtc works for me

Comment (GeorgeChatzigiannis) [reply admin/views/pupil-register/pupils-list.ejs]: @jon-shipley I will amend all the component clones that have received new markup and style on the next PR as agreed.

**Pull request merged**